fli-iam/shanoir-ng
# Shanoir issue#2284: solr filter importDate

How to test:
Go to solr page
Open "import date" filter and try it out
The solr result should be filtered accordingly

An endDate value set before the startDate should display an error:
![image](https://github.com/user-attachments/assets/0952fb46-46ee-47c0-9257-92e840823e12)
